src/wallet/wallet.cpp

```diff
@@ -135,8 +134,7 @@ bool RemoveWallet(const std::shared_ptr<CWallet>& wallet, std::optional<bool> lo
         vpwallets.erase(i);
     }
 
-    assert(::coinJoinWalletManager != nullptr);
-    ::coinJoinWalletManager->Remove(name);
+    wallet->coinjoin_loader().RemoveWallet(name);
```

Comment:
feels like it should be called something different if the "loader" also handles removal :)

Reply:
there's already too many managers: CoinJoinWalletManager,  CCoinJoinClientManager, CCoinJoinClientManager, CCoinJoinClientQueueManager, CCoinJoinBaseManager...

it can be CCoinJoinWalletManager but this name is busy already. Let's keep coinjoin_loader to avoid confusion.